src/control/cmd/dmg/main.go

```diff
@@ -211,6 +214,24 @@ and access control settings, along with system wide operations.`
 			common.ScrubProxyVariables()
 		}
 
+		if opts.LogFile != "" {
+			f, err := common.AppendFile(opts.LogFile)
+			if err != nil {
+				return errors.WithMessage(err, "create log file")
+			}
+			defer f.Close()
```

Comment:
Just for my understanding.
I have seen the same code in daos-agent, but I don't figure out how the output is still valid after this function call: The output seems to not be open in the NewErrorLogger ?

Reply:
The deferred call happens just before the command handler function returns, so we only close the log file after the command is executed.